lib/core/routes.dart

```diff
@@ -26,10 +28,10 @@ class AppPages {
     //   name: Routes.homeScreen,
     //   page: () => const HomeScreen(),
     // ),
-    // GetPage(
-    //   name: Routes.loginScreen,
```

Comment:
Please reformat the code and removed unused code

Reply:
Removed the commented lines.. And formatted with VS code default formatting